package.json

```diff
@@ -44,25 +45,9 @@
     "version"
   ],
   "license": "MIT",
-  "main": "index.js",
-  "nyc": {
-    "include": [
-      "lib/**/*.js",
-      "index.js"
-    ],
-    "reporter": [
-      "json",
-      "text",
-      "html"
-    ],
-    "all": true
-  },
+  "exports": "./index.js",
```

Comment:
i know `main` isnt strictly needed after switching to `exports`, but i've hit enough snags with removing it from other projects, that i think it is still best to keep around for backward compatibility for now. lets have both `exports` and `main` set to the same value

Reply:
Yea, better keep it safe